cachi2/core/package_managers/bundler/parser.py

```diff
@@ -79,6 +81,33 @@ class GitDependency(_GemMetadata):
     url: AcceptedUrl
     ref: AcceptedGitRef
 
+    @property
```

Comment:
A super non-significant, non-blocking nitpick, please apply only if you end up doing another revision: this won't change, so could be a cached_property.

Reply:
no worries, it is just a matter of seconds :smile: